src/pages/LocalScript.tsx

```diff
@@ -335,6 +341,48 @@ export function LocalScript({
 
   const [result, setResult] = useState<RunResult>({ type: "init" });
 
+  useEffect(() => {
+    // TODO: There exists some strangeness (maybe a bug somewhere):
+    // - Open in local dynamo a script that is saved from one of the "Graphs provided by Autodesk" e.g. cloud models.
+    // - Open the script here, but from the "Graphs provided by Autodesk" section. e.g. open a new cloud copy.
+    // The local script and the cloud model is now linked, and running the cloud model will run the local script.
+    // The names in dynamo and dynamo extensions are however out of sync.
+    // **May be related to the name variable that is rendered below.**
```

Comment:
Would love to discuss this with someone

Reply:
Video of what happens: 
https://github.com/user-attachments/assets/ade391b9-4f0c-425b-8b96-9b1bcc09fb04